hasher-matcher-actioner/terraform/main.tf

```diff
@@ -51,7 +51,8 @@ module "pdq_signals" {
   }
   threat_exchange_data = {
     bucket_name        = module.hashing_data.threat_exchange_data_folder_info.bucket_name
-    pdq_data_file_key  = local.pdq_data_file_key
+    pdq_key_suffix     = local.pdq_key_suffix
+    data_folder        = module.hashing_data.threat_exchange_data_folder_info.key
```

Comment:
See note about folder being a misnomer

Reply:
Does the key variable need to be renamed? These are the inconsistencies I was flagging above.

Also should pdq_key_suffix be file_extension? Folder, files or key prefix, key suffix?

Reply:
I think `pdq_file_extention` makes sense along with `data_folder`. Will change